File: Java_Advanced/gsapi/src/main/java/br/com/fiap/gs/gsapi/dto/external/GoogleGeocodingLocationDTO.java
package br.com.fiap.gs.gsapi.dto.external;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;

@JsonIgnoreProperties(ignoreUnknown = true)
public class GoogleGeocodingLocationDTO {

    @JsonProperty("lat")
    private double latitude;

    @JsonProperty("lng")
    private double longitude;

    public GoogleGeocodingLocationDTO() {
    }

    // Getters e Setters
    public double getLatitude() {
        return latitude;
    }
    public void setLatitude(double latitude) {
        this.latitude = latitude;
    }
    public double getLongitude() {
        return longitude;
    }
    public void setLongitude(double longitude) {
        this.longitude = longitude;
    }
}

File: Advanced_Business_Development_with.NET/gsApi/gsApi/dto/response/GoogleGeocodingLocationDto.cs
// File: gsApi/dto/response/GoogleGeocodingLocationDto.cs
using System.Text.Json.Serialization;
namespace gsApi.dto.response
{
    public class GoogleGeocodingLocationDto
    {
        [JsonPropertyName("lat")]
        public double Latitude { get; set; }

        [JsonPropertyName("lng")]
        public double Longitude { get; set; }
    }
}